### PR TITLE
v99.0.0

### PR DESCRIPTION
**Pull request by galiak11**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,68 @@
+# 99.0.0
+
+In this major release we deleted and deprecated numerous APIs in ButtonBar, FeatureHighlight, List,
+Tabs, TextFields and Flexible Header. We fixed layout issue in Dialogs and BottomAppBar and added
```
Comment (jverkoey) [inline CHANGELOG.md:4]: issues*

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+support for non-transient Snackbars.
```
Comment (jverkoey) [inline CHANGELOG.md:5]: Please add a blurb about the new support for non-transient Snackbars. @yarneo for details.

File: CHANGELOG.md (hunk 1 above)
Comment (galiak11) [reply CHANGELOG.md:4]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (galiak11) [reply CHANGELOG.md:5]: Done

Comment (jverkoey) [reply CHANGELOG.md:5]: Please include this in a “New features” section and include a code snippet in either objective c or swift.

Comment (jverkoey) [reply CHANGELOG.md]: Please include this in a “New features” section and include a code snippet in either objective c or swift.

**Review by jverkoey — APPROVED** [review]

**Review by andrewoverton — APPROVED** [review]
LGTM ~pending any additional feedback from @jverkoey~

**Pull request merged**